### ISSUE TITLE
Che installation in Docker Desktop (Windows) - how to access Che dashboard

### ISSUE THREAD

**Issue by filipthoen2** [eclipse/che#21066]
### Summary

Hey CHe community!

First time user here - I installed Che in Docker Desktop (Windows) today & after a few misfires (the 'PostgreSQL pod bootstrap' post installation step bailing due to:

Cause: Error: Eclipse Che operator failed, reason: InstallOrUpdateFailed, message: no matches for kind "Issuer" in version "cert-manager.io/v1". Consider increasing error recheck timeout with --k8spoderrorrechecktimeout flag.) I was able to complete the installation successfully & it looks like Che is fully up & running.

I followed the (brief) instructions on : https://www.eclipse.org/che/docs/che-7/installation-guide/installing-che-on-docker-desktop/

I have 2 questions:
1. which is the URL to access Che (the docs don't mention how to get this)?
2. how do I add a 'workspace' as the chectl workspace commands seem to have been retired (the screen recording video on 'https://www.eclipse.org/che/docs/che-7/installation-guide/installing-che-locally/' is out of date & misleading...)?

Rgd #1., I did:

C:\Users\filip>chectl server:status
Eclipse Che Version    : 7.43.0-SNAPSHOT
Eclipse Che Url        : https://192.168.56.1.nip.io
OpenShift OAuth enabled: No

Opening https://192.168.56.1.nip.io in my browser (Chrome), leads to:

   _This site can’t be reached192.168.56.1.nip.io unexpectedly closed the connection_

I tried 'telnet 192.168.56.1.nip.io 443' which successfully opens a TPC/IP connection - so someone is listening on the other end.

I also tried 'chectl dashboard:open', which spawns a browser tab: https://192.168.56.1.nip.io/dashboard/, which gives the same web server error.

Rgd. #2, I assume you use the Che dashboard for this, which I can't open (yet)...

Version info below.

Thanks for all your help!

Filip

### Relevant information

OS: Windows 10 Pro
Docker Desktop:  20.10.12
Kubectl: 1.22.5
Chectl: chectl/7.42.0 win32-x64 node-v12.22.9

**Comment by chess3cake** [comment]
I have same error in minikube.But if I specify --version=7.41.0, everything is ok
Just like this:
chectl server:deploy --platform=minikube --version=7.41.0

**Comment by filipthoen2** [comment]
Thanks, chess3cake! The deployment goes much smoother & I get less errors during the deploy step.

I noticed that in my previous deploys, chectl ended up with the following URL:

https://192.168.56.1.nip.io/dashboard/

This doesn't make sense -- 192.168.56.1 was actually the default address of the VirtualBox network bridge I've installed on my laptop. I've disabled the network in Windows, hoping chectl would actually pick a physical network controller with a real IP address. And it did! Now it uses 192.168.1.204 which is the DHCP address of my phys network card:

C:\Users\filip>chectl server:status
Eclipse Che Version    : 7.41.0
Eclipse Che Url        : https://che-eclipse-che.192.168.1.204.nip.io
OpenShift OAuth enabled: No

However, I could still not connect in a browser, and Chrome gives the error (even after importing the Che certificate):

_This site can’t be reached che-eclipse-che.192.168.1.204.nip.io unexpectedly closed the connection._

I tried curl & the web serer seems to respond, but likely fails due to certificate issue:

curl -v https://che-eclipse-che.192.168.1.204.nip.io
*   Trying 192.168.1.204:443...
* Connected to che-eclipse-che.192.168.1.204.nip.io (192.168.1.204) port 443 (#0)
* schannel: disabled automatic use of client certificate
* schannel: ALPN, offering http/1.1
* schannel: failed to receive handshake, SSL/TLS connection failed
* Closing connection 0
curl: (35) schannel: failed to receive handshake, SSL/TLS connection failed

I tried curl with options to ignore SSL warnings (-k or -insecure) but I get the same error.

What does the curl command produce on your machine?

Thanks for all your help, much appreciated!

Filip
